libraries/engage/app-rating/selectors/appRating.js

```diff
@@ -0,0 +1,8 @@
+import { createSelector } from 'reselect';
+
+/**
+ * Selects the app rating information.
+ * @param {Object} state The current state.
+ * @returns {Object} The app rating information.
+ */
+export default createSelector(state => state.appRating, rating => rating);
```

Comment:
```suggestion
export const getAppRatingState => state.appRating;
```
reselect won't help here since and will introduce more overhead then gains